### DISCUSSION TITLE
WHEN new release

### DISCUSSION THREAD

**Discussion by Kolek1995** [redphx/better-xcloud#155] [General]
Hi can i know WHEN new realease coming and what change IT will be?sorry for my english

**Comment by KingNothing81** [comment]
Not that the script needs an update as everything still works fine ... but I hope @redphx isn't retiring? ;)

**Comment by redphx** [comment]
You can test the dev version here: https://github.com/redphx/better-xcloud/raw/feature/shortcuts/better-xcloud.user.js  

This adds a new "controller shortcuts" option in the Settings.  
While playing: hold Home + RB to take screenshot, or Home + Select to toggle Stats bar. Home = the button on your controller that brings up the Xbox sidebar menu. More shortcuts will be added later.  

I'm also working on adding the Remote Play feature. I was able to make it work, but need some more time to properly implement it to Better xCloud.

**Reply by KingNothing81** [reply]
> Home = the button on your controller that brings up the Xbox sidebar menu. More shortcuts will be added later.

Sounds like a great addition. But unfortunately no button on my Shield TV controller will act like this and bring up the Xbox sidebar when using xCloud with Kiwi on Android TV. And I think many controllers behave like this on Android TV: That pressing the middle home button will just bring you to the Android home screen.

So maybe it would be possible to make the shortcut feature more flexible, by allowing us to set another button instead of Home to trigger the shortcuts? For example the Start/Menu button?

**Reply by redphx** [reply]
Test your controller in https://hardwaretester.com/gamepad  
The browser can only detect 17 buttons on the controller. The home button is mapped to B16.  
I can make it use other buttons, but not right now.

**Reply by KingNothing81** [reply]
Did you test the controller shortcut feature on an Android (TV) device? And does it work for you? Which controller do you use?

As you mention in your changelog, the button doesn't exist on all controllers or is blocked by some internal remapping. Like Nivida does on Shield TV, remapping the Xbox button on Xbox controllers to go to the home screen. So it can't be used to open the Xbox sidebar when using xCloud. 

So I think you should consider using a button that is present/working on all controllers and platforms. So the shortcurt feature will be useable for every user.
